lib/rules/prop-types.js

```diff
@@ -301,6 +301,17 @@ module.exports = {
     }
 
     /**
+     * Removes quotes from around an identifier.
+     * @param {string} the identifier to strip
+     */
+    function stripQuotes(string) {
+      if (string[0] === '\'' || string[0] === '"') {
```

Comment:
Will this string always have matching quotes? or would it be better to do an unconditional replace that looks for a starting `'`/`"` and a matching token at the end?